tests/Tests/_VerifySnapshots/FunctionScoreQueryUsageTests.VerifyDescriptorJson.verified.txt

```diff
@@ -1,7 +1,7 @@
 {
   query: {
     function_score: {
-      boost: 1.1,
```

Comment:
NETFX is not fully IEEE compliant which messes with our roundtrip serialization tests. Particularily `float` values seem to cause problems. As a workaround I changed these tests to use values that can be represented as integers.

More details: https://github.com/dotnet/runtime/issues/435#issuecomment-560503912